### PR TITLE
Improve FFI part a bit

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,9 +30,30 @@ In FFI (`-m sys`) mode, `gir` generates as much as it can. So in this mode, the
...
+[options]
+girs_dir = "../gir-files"
+library = "GtkSource"
+version = "3.0"
+min_cfg_version = "3.0"
+target_path = "."
+work_mode = "sys"
+```
+
 And that's all! Neither `GtkWidget` nor `GtkWindow` (alongside with their functions) will be generated.
```
Comment (EPashkin) [inline README.md]: Insert place slightly wrong, this is `ignore` part.

File: README.md [hunk 2]
```diff
@@ -30,9 +30,30 @@ In FFI (`-m sys`) mode, `gir` generates as much as it can. So in this mode, the
...
+You also need to add any needed external libraries in the "external_libraries" parameter. Example:
+
+```toml
+external_libraries = [
+   "GLib",
+   "GObject",
+]
+```
+
+You can specify a few other options:
```
Comment (EPashkin) [inline README.md]: IMHO also better add "Part of its can overridden from command line"

**Comment by EPashkin** [comment]
Thanks

File: README.md (hunk 2 above)
Comment (GuillaumeGomez) [reply README.md]: I didn't know, how?

Comment (EPashkin) [reply README.md]: Also not sure :(

Comment (GuillaumeGomez) [reply README.md]: Haha! Then let's not mention it. I'll update the other issues. ;)

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by EPashkin** [comment]
Thanks

**Pull request merged**